lib/crypto/tasks.py

```diff
@@ -24,43 +24,51 @@ def __exit__(self, type, value, traceback):
 
 
 @task
-def sign_addons(addon_ids, **kw):
+def sign_addons(addon_ids, force=False, **kw):
```

Comment:
The chunking is done in the `process_addons` management command (see above comment). Do you think we should replicate it here, in case this task is used directly? (for example by the `sign_addons` command with a huge list of IDs?)